UI/GameSettingsScreen.cpp

```diff
@@ -1164,8 +1165,10 @@ void GameSettingsScreen::onFinish(DialogResult result) {
 	KeyMap::UpdateNativeMenuKeys();
 
 	// Wipe some caches after potentially changing settings.
+#ifndef HAVE_LIBNX
```

Comment:
Was there a problem this caused?

This will prevent some features from working, like post-processing shaders and certain settings changes.

-[Unknown]

Reply:
IIrc some stuff triggered a context reset(?) and that caused a crash